examples/10-vaex-pipeline/tasks/fit.py

```diff
@@ -0,0 +1,33 @@
+import vaex, json
+from vaex.ml.sklearn import IncrementalPredictor
+from sklearn.naive_bayes import CategoricalNB
+from sklearn.metrics import accuracy_score
+
+from cowait import Task
+from utils import get_outpath, get_classes
+
+class Fit(Task):
+    async def run(self, inpath, alpha, size):
+        model      = CategoricalNB(alpha=alpha)
+        vaex_model = IncrementalPredictor(features=['PULocationID', 'dayofweek', 'hour'],
+                                          target='DOLocationID',
+                                          model=model, 
+                                          batch_size=100_000, 
+                                          partial_fit_kwargs={'classes': get_classes()}
+                                         )
+
+        df = vaex.open(inpath)
+        vaex_model.fit(df=df, progress=True)
+
+        df  = vaex_model.transform(df=df)
+        acc = accuracy_score(y_true=df['DOLocationID'].values,
+                             y_pred=df['prediction'].values)
```

Comment:
Hi, 

The calculation of accuracy is a possible suspect regarding the memory issues: since the data read in memory. 

One day we hope such functions will work out of core in sklearn via vaex, but until than, perhaps you can do something like 
```
acc = df.sum(df['prediction'] == df['DOLocationID']) / len(df)
```

Reply:
Ah, of course. Thanks